community/leighphan.md

```diff
@@ -0,0 +1,7 @@
+# Leigh Phan
+
+Hi! 👋 I'm Leigh and I like designing, building community, and learning to code. My nephew Cory the corgi is more famous than you.
+
+- Twitter: [@leighphan](https://www.twitter.com/leighphan)
+- Website: [hello](http://leighphan.github.io/hello)
```

Comment:
I think you can just set the URLs instead of using the markdown syntax, it will be easier to parse later with JavaScript